lib/models/application.coffee

```diff
@@ -874,6 +874,62 @@ getApplicationModel = (deps, opts) ->
 				body: body
 		.asCallback(callback)
 
+	###*
+	# @summary Configure a specific application to stop tracking the latest available release
+	# @name disableTrackLatestRelease
+	# @public
+	# @function
+	# @memberof balena.models.application
+	#
+	# @description The application's current release will not be updated by successfully built releases.
+	#
+	# @param {String|Number} nameOrId - application name (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# balena.models.application.disableTrackLatestRelease('MyApp').then(function() {
+	# 	...
+	# });
+	#
+	# @example
+	# balena.models.application.disableTrackLatestRelease(123).then(function() {
+	# 	...
+	# });
+	#
+	# @example
+	# balena.models.application.disableTrackLatestRelease('MyApp', function(error) {
+	# 	if (error) throw error;
+	# 	...
+	# });
+	###
+	exports.disableTrackLatestRelease = (nameOrId, callback) ->
```

Comment:
I think that the name of the method doesn't fully describe what it's doing under the hood.
It sounds like it should just pin the application to its current release but the implementation actually pins it to the latest release instead, possibly resulting the whole fleet to download updates if a successful release was published in between two calls of this method.